app/src/main/java/com/infomaniak/mail/utils/ConfettiUtils.kt

```diff
@@ -113,13 +156,13 @@ object ConfettiUtils {
             displayConfetti(config, sourceRight, container)
         }
 
-        when ((0..2).random()) {
-            0 -> displayTada()
-            1 -> displaySnow()
-            else -> if (isInPortrait()) {
-                displaySingleGeneva()
-            } else {
-                displayDoubleGeneva()
+        if (matomoValue == "Tablet") {
```

Comment:
Not sure we should do conditions on matomo values

Reply:
I agree